lib/nginx.py

```diff
@@ -32,7 +32,7 @@ def get_path_config():
     Default for satisfy is any
     """
 
-    location_template = """
+    location_template = r"""
```

Comment:
Changing a normal multi-line string to a raw string is a functional change. Why is this done along with updating the dependencies? And, what is the rationale for this change?

Reply:
no longer in mr.